client/src/generated/graphql.tsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable */
 import gql from 'graphql-tag';
-import * as ApolloReactCommon from '@apollo/react-common';
-import * as ApolloReactHooks from '@apollo/react-hooks';
+import * as ApolloReactCommon from '@apollo/client';
+import * as ApolloReactHooks from '@apollo/client';
```

Comment:
you can replace the 2 import with `import { ApolloProvider } from '@apollo/client';`

Reply:
@pradel but pipelines are failing now 😬 

Actually if I run yarn generate from `client` it automatically updates to :

```
import * as ApolloReactCommon from '@apollo/react-common';
import * as ApolloReactHooks from '@apollo/react-hooks';
```

Reply:
my bad I didn't see it was from the generated file 🤦